src/main/java/org/opentripplanner/netex/mapping/WgsCoordinateMapper.java

```diff
@@ -21,10 +22,19 @@ static WgsCoordinate mapToDomain(SimplePoint_VersionStructure point) {
     }
```

Comment:
The Javadoc for the mapToDomain method is obsolete, this PR gives a good opportunity to update it.
- You can explain here that we need to convert from the LON/LAT convention in GML to the LAT/LON convention in WGS since this is a usual source of bug/confusion.
- You can also describe here the fallback logic you implemented.

Reply:
@giuseppesaraceno can you update the doc?